scripts/zombienet/all-dev-local.toml

```diff
@@ -78,7 +78,7 @@ id = 2087
 
 [[parachains.collators]]
 command = "../../target/release/composable" # nix build .#composable-node
-name = "alice" # must be 100 alice and only alice because our chain is hardcoded with it, prevent other nodes to be alice
+name = "alice"                              # must be 100 alice and only alice because our chain is hardcoded with it, prevent other nodes to be alice
```

Comment:
```suggestion
name = "alice" # must be 100 alice and only alice because our chain is hardcoded with it, prevent other nodes to be alice
```

Reply:
it is by `$ nix run .#fmt` . Fixing our CI fmt is other PR. So I run it and if it does what it does - this is okey.

So fixing `$ nix run .#fmt` is not part of this PR, but applying official format is not blocker either.

Agree it is ugly, but I am for consistency instead of ad hoc manual fixes.